src/eth/TransactionManager.js

```diff
@@ -210,6 +210,23 @@ class Tracker {
     return txs[0];
   }
 
+  clearExpiredTransactions() {
+    each(this._transactions, (txList, key) => {
```

Comment:
I realized I was removing the key for multi-transaction operations before the other tx's could run, so now we loop over each key and check the transactions individually.